packages/server-side-graphql-client/README.md

```diff
@@ -134,6 +134,36 @@ const addUser = async userInput => {
 addUser({ name: 'keystone user', email: 'keystone@test.com' });
 ```
 
+For relationship fields, use a `<relatedField>: { connect: { id: <idValue> } }` construct. Example:
```

Comment:
```suggestion
Note: The `item` property is a graphQL create input. For Relationship fields it can contain nested mutations with create and connect operations. For examples see the [Relationship API documentation](/packages/fields/src/types/relationship/#create-and-append-a-related-item).
```